src/components/ProcessCreate/CreationProgress.tsx

```diff
@@ -43,24 +43,28 @@ export const CreationProgress = ({ error, sending, step }: CreationProgressProps
   }, [step])
 
   return (
-    <Stack direction='column' gap={4}>
-      <Text>{t('process_create.creation_steps_description')}</Text>
-      <Heading as='h2' size='md' textAlign='center'>
-        {t('process_create.creation_steps_title')}
-      </Heading>
+    <Stack direction='column' gap={5} p={0}>
+      <Box bgImage='url(/assets/spreadsheet-confirm-modal.png)' height='150px' bgSize='cover' borderRadius='lg' />
+      <Text mb={6} textAlign='center'>
+        {t('process_create.creation_steps_description')}
+      </Text>
       <List spacing={3}>
         {Object.keys(labels).map((key) => (
-          <ListItem key={key}>
-            <ListIcon
-              as={steps[key as keyof CreationStepsState] ? AiFillCheckCircle : AiFillCloseCircle}
-              fontSize={22}
-            />
-            {labels[key]}
+          <ListItem key={key} display='flex'>
+            {steps[key as keyof CreationStepsState] ? (
+              <ListIcon as={AiFillCheckCircle} fontSize={22} />
+            ) : (
+              <Spinner boxSize={5} />
```

Comment:
Final result shows more space than spinner.

![imatge](https://github.com/vocdoni/ui-scaffold/assets/153305/ed97f887-6008-479e-a9c6-73178395f733)

Reply:
Also, if there's an error, the spinners should not be visible, and show the old "close" icon, maybe.
![imatge](https://github.com/vocdoni/ui-scaffold/assets/153305/93cc2647-836c-449d-a4ce-29281264e8ac)